### ISSUE TITLE
sorting is broken in ie11 && FF when scrolled down

### ISSUE THREAD

**Issue by bunkscene** [jasonslyvia/react-anything-sortable#84]
Hey @jasonslyvia,

We noticed an issue when a sortable container has containment=true and is way down in a page where the scrolltop is being used.  In IE11 and FF, document.body.scrollTop is not supported.  This causes a downstream affect where the sortable item cannot be dragged.

```
componentDidUpdate() {
  const container = ReactDOM.findDOMNode(this);
  const rect = container.getBoundingClientRect();

  this._top = rect.top + document.body.scrollTop;
  this._left = rect.left + document.body.scrollLeft;
  this._bottom = this._top + rect.height;
  this._right = this._left + rect.width;
}
```

**Comment by jasonslyvia** [comment]
Thanks for reporting, I 'll see into it soon.

**Comment by jasonslyvia** [comment]
I've published version `1.7.1` in case you need it.

**Comment by bunkscene** [comment]
Great, thanks for the quick fix!  However, the new code has a bug in it as well.  Index.js:18 has a reference to window.  When server side rendering, this fails.  Can you add a check for that?
